app/assets/stylesheets/05-page.scss

```diff
@@ -0,0 +1,35 @@
+.page_container {
```

Comment:
What's the significance of the name `05-page`?

Reply:
In this version of Rails, there is a SASS compiler that runs the .scss all together as the CSS sheets. We were taught to create a new .scss file per component to help with organization + number it for ordering.